pkg/models/model.go

```diff
@@ -123,9 +131,33 @@ func (m *Model) setNodeMap() error {
 				NetLayers:     netLayers,
 				CapacityBlock: cb.Name,
 			}
+
+			region, ok := metadata["region"]
+			if !ok {
+				region = "none"
+			}
+			if _, ok := regions[region]; !ok {
+				regions[region] = make(map[string]string)
+
+			}
+			regions[region][name] = name
```

Comment:
Super Nit: "Upserting" into a map usually looks a little more like this. Slightly more efficient but +1 line.

```suggestion
			r, ok := regions[region]
			if !ok {
			    r = make(map[string]string)
			    regions[region] = r
			}
			r[name] = name
```